### DISCUSSION TITLE
How to enable HELI MIX

### DISCUSSION THREAD

**Discussion by dimask1971** [EdgeTX/edgetx#2778] [Q&A]
How to request firmware 2.8 with HELI MIX enabled? To set up a helicopter with a flybar.

**Comment by pfeerick** [comment]
Hopefully in the next few days, since 2.8 hasn't been released, so not doing any custom builds yet.

**Comment by dimask1971** [comment]
In version 2.8, the HELP SETUP menu item is unavailable. In the current and future versions, this menu item is not available by default? Need to assemble the firmware yourself with HELI SETUP enabled?

**Reply by gagarinlg** [reply]
You can either build yourself or ask for help in the EdgeTX discord.

**Reply by pfeerick** [reply]
Check out #2362
